spec/robots/dor_repo/speech_to_text/stt_create_spec.rb

```diff
@@ -44,24 +46,31 @@
     let(:message_body) { { id: job_id, druid:, media: }.merge(Settings.speech_to_text.whisper.to_h).to_json }
 
     before do
-      allow(aws_client).to receive(:send_message).with({ queue_url: Settings.aws.speech_to_text.sqs_todo_queue_url, message_body: }).and_return(true)
+      allow(aws_batch_client).to receive(:submit_job).with({ props: { message_body: } }).and_return(true)
```

Comment:
i think i'm slightly confused... where does `props` come from in this structure?  i can't seem to find it in any of the calling code, in either the app or the specs.  (definitely not something i'd hold up the PR over, i suspect i'm missing something, and everything else makes sense to me)

Reply:
Good eye, that appears to fluff left over from me trying to get the tests set up right. I'm removing it to avoid further confusion.